### DISCUSSION TITLE
Update Chocolatey package

### DISCUSSION THREAD

**Discussion by yan12125** [Tyrrrz/LightBulb#317] [General]
There is a Chocolatey package https://community.chocolatey.org/packages/lightbulb, but it's quite out-dated. Could you consider updating it?

In https://github.com/Tyrrrz/LightBulb/issues/186#issuecomment-748525229, you mentioned:

> Chocolatey deployments got annoying to orchestrate properly in CD, so I stopped doing them. There is, however, a WinGet package that is maintained by the community: https://github.com/microsoft/winget-pkgs/tree/master/manifests/Tyrrrz/LightBulb
> 
> If someone also wants to take Chocolatey over, I can give the publishing rights to the package (assuming it's possible, I don't know).

If you wish, I'm willing to take it over. I may maintain the nuspec and relevant files in my own repo (i.e., not via CD in this repo).

FWIW, there is a "Manage Maintainers" button in my packages. Apparently that can be used to add co-maintainers.

**Comment by Tyrrrz** [comment]
Hi @yan12125.

Sure, let me know your choco username.

**Reply by yan12125** [reply]
Here it is, thanks: https://community.chocolatey.org/profiles/yan12125

**Reply by Tyrrrz** [reply]
Added you as a maintainer

**Reply by yan12125** [reply]
Thanks! Somehow pushing the updated package fails:
```
Chocolatey v2.3.0
Attempting to push lightbulb.2.5.3.nupkg to https://push.chocolatey.org/
回應狀態碼未表示成功: 403 (The specified API key does not provide the authority to push packages. This package may already exist and be owned by a different user. It could also be hidden. Try going directly to https://community.chocolatey.org/packages/'packageName' to determine mai)。
```
I didn't see my name on https://community.chocolatey.org/packages/lightbulb either. Could you double check?

**Reply by Tyrrrz** [reply]
![image](https://github.com/Tyrrrz/LightBulb/assets/1935960/3b5a8338-124a-4c8e-bfa3-6cc51378d6dc)

Maybe you need to accept an invitation of some sorts? Or does Chocolatey need to approve this?

**Reply by yan12125** [reply]
Hmm, I didn't see an invitation. Let me try to ask Chocolatey maintainers about how to handle this.

**Reply by yan12125** [reply]
With kind help from gep13 on Chocolatey Discord server, I got my confirmation email, became a co-maintainer and uploaded 2.5.3. Now waiting moderation :)

**Reply by Tyrrrz** [reply]
Awesome :)
